src/lib/Resolver/DomainContentResolver.php

```diff
@@ -132,14 +132,6 @@ public function resolveMainUrlAlias(Content $content)
         return isset($aliases[0]->path) ? $aliases[0]->path : null;
     }
 
-    /**
-     * @deprecated since v3.0, use ItemResolver::resolveItemFieldValue() instead.
-     */
-    public function resolveDomainFieldValue(Content $content, $fieldDefinitionIdentifier, $args = null)
```

Comment:
Please look at https://github.com/ibexa/graphql/blob/main/src/bundle/Resources/config/services/resolvers.yaml#L35
I think this maybe the only case we have here, but we need to check if other removed methods are not used anywhere.